themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```
@@ -12,6 +22,20 @@ sessionsHtmlTitle=Sessies
 accountManagementTitle=Keycloak Accountbeheer
 authenticatorTitle=Authenticator
 applicationsHtmlTitle=Toepassingen
+
+accountManagementWelcomeMessage=Welkom bij Keycloak Account Management
+personalInfoIntroMessage=Beheer uw basisgegevens
+accountSecurityTitle=Account Beveiliging
+accountSecurityIntroMessage=Beheer uw wachtwoord en account toegang
+applicationsIntroMessage=Volg en beheer uw app-toestemming voor toegang tot uw account
+resourceIntroMessage=Deel uw bronnen met uw team
+passwordLastUpdateMessage=Uw wachtwoord was veranderd op
+updatePasswordTitle=Update Wachtwoord
```

A more 'Dutch' approach would be:
```suggestion
updatePasswordTitle=Wachtwoord Bijwerken
```